bin/compile

```diff
@@ -585,12 +585,25 @@ then
   echo -e "web: $(basename $name)" >> $build/Procfile
 fi
 
+cd $build
 rm -rf $build/.heroku
 
 mkdir -p $build/.profile.d
 echo 'PATH=$PATH:$HOME/bin' > $build/.profile.d/go.sh
 
+if test "${GO_INSTALL_TOOLS_IN_IMAGE}" = "true"
```

Comment:
Minor nitpick: consider using the presence of the env var as a trigger for the behaviour (ie, don't consider the value at all) to avoid confusion where someone uses `GO_INSTALL_TOOLS_IN_IMAGE=1` and then has a confusing follow on experience.